### ISSUE TITLE
Add FORMAT macro

### ISSUE THREAD

**Issue by oubiwann** [lfex/cl#18]
This can wrap `lfe_io:format` and be used simply as `format` when developers include `lfe/include/cl.lfe`.

**Comment by rvirding** [comment]
Yes, I was thinking we would need to include some macros. Unfortunately they won't be called with a `cl:`prefix so it will be `(format ... )` not `(cl:format ...)`. Must think about this.

**Comment by rvirding** [comment]
I have just realised that you can define macros with a `cl:` prefix so we can make it look consistent.
